src/main/java/school/lemon/changerequest/java/introduction/hw1/Task8.java

```diff
@@ -2,6 +2,15 @@
 
 public class Task8 {
     public static int calculateSum(int N) {
-        return 0;
+        int power = 2;
+        int base;
+
+        int sum = (int) Math.pow(N, power);
```

Comment:
`int sum = (int) Math.pow(N, power);`, `sum + (int) Math.pow((2 * N), power);` can be moved inside the loop.